evennia/contrib/rpg/rpsystem/rpsystem.py

```diff
@@ -439,7 +442,7 @@ def parse_sdescs_and_recogs(
                 (re.search(rquery, text, _RE_FLAGS), obj, text) for obj, text in candidate_map
             )
             # filter out any non-matching candidates
-            bestmatches = [(obj, match.group()) for match, obj, text in matches if match]
+            bestmatches = [(obj, mtch.group()) for mtch, obj, text in matches if mtch]
```

Comment:
avoid overlapping the keyword `match` (which wasn't present when this was originally written)

Reply:
this wasn't supposed to be a review, just a comment, but i fatfingered it 😩